decompiler/pipeline/controlflowanalysis/expression_simplification/rules/sub_to_add.py

```diff
@@ -0,0 +1,27 @@
+from decompiler.pipeline.controlflowanalysis.expression_simplification.rules.rule import SimplificationRule
+from decompiler.structures.pseudo import BinaryOperation, Constant, Expression, Operation, OperationType
+
+
+class SubToAdd(SimplificationRule):
```

Comment:
I originally did it this way because it would have simplified a different stage which would have done integer factorization. However, I decided to not implement this for now. Fixed with b62a4232.